packages/conf-pkg-config/conf-pkg-config.2/opam

```diff
@@ -32,7 +32,7 @@ depexts: [
   ["pkgconf"] {os = "freebsd"}
   ["pkgconf-pkg-config"] {os-distribution = "rhel" & os-version >= "8"}
   ["pkgconf-pkg-config"] {os-distribution = "centos" & os-version >= "8"}
-  ["pkg-config"] {os-distribution = "cygwinports"}
+  ["system:pkgconf"] {os = "win32" & os-distribution = "cygwinports"}
```

Comment:
So only win64 uses the other repo?

Reply:
@mseri : confusingly 64 bit windows als has os=win32. Here is what `opam var` gives on 64 bit cygwin:
```
arch              x86_64                                        # Inferred from system
opam-version      2.0.8                                         # The currently running opam version
os                win32                                         # Inferred from system
os-distribution   cygwinports                                   # Inferred from system
os-family         windows                                       # Inferred from system
```
Note the value of the arch variable. This i686 on 32 bit cygwin.

Also note that this is the MinGW patched opam, not the plain cygwin opam, but I guess this would be the same.

Anyway I would also recommend to remove the `os = "win32"` test since it is confusing and superflous. `os-distribution = "cygwinports"` implies `os = "win32"`.